Mathlib/Algebra/Group/Hom/Defs.lean

```diff
@@ -108,6 +110,9 @@ structure AddHom (M : Type*) (N : Type*) [Add M] [Add N] where
   /-- The proposition that the function preserves addition -/
   protected map_add' : ∀ x y, toFun (x + y) = toFun x + toFun y
 
+/-- `M →ₙ+ N` denotes the type of addition-preserving maps from `M` to `N`. -/
+infixr:25 " →ₙ+ " => AddHom
```

Comment:
Can you also add this new notation in the `## Notations` section in the module doc (line 34 or so of this file)? Thanks.